package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-application",
-  "version": "2.7.0",
+  "version": "2.7.1",
```

Comment:
This is a new feature for me, so should be a minor version increase

Reply:
fair point, will bump it.